docs/running-on-kubernetes.md

```diff
@@ -768,6 +768,22 @@ from the other deployment modes. See the [configuration page](configuration.html
     <code>myIdentifier</code>. Multiple node selector keys can be added by setting multiple configurations with this prefix.
   </td>
 </tr>
+<tr>
+  <td><code>spark.executorEnv.[EnvironmentVariableName]</code></td> 
```

Comment:
this is already doc'd in `docs/configuration.md` -- please remove this duplicate config from `docs/running-on-kubernetes.md` and reference it in the `spark.kubernetes.driverEnv.[EnvironmentVariableName]` section below with language something like `For executors, use the generic spark.executorEnv.[EnvironmentVariableName] which works across cluster managers`